src/fragments/lib-v1/restapi/flutter/fetch.mdx

```diff
@@ -53,19 +77,47 @@ app.get('/todo', function(req, res) {
 
 Then you can use query parameters in your path as follows:
 
-```dart
-Future<void> getTodo() async {
-  try {
-    const options = RestOptions(
-        path: '/todo',
-        queryParameters: {'q' : 'test'},
-    );
-    final restOperation = Amplify.API.get(restOptions: options);
-    final response = await restOperation.response;
-    print('GET call succeeded');
-    print(String.fromCharCodes(response.data));
-  } on ApiException catch (e) {
-    print('GET call failed: $e');
+
+<BlockSwitcher>
+
+<Block name="Stable (Mobile)">
+
+  ```dart
+  Future<void> getTodo() async {
+    try {
+      const options = RestOptions(
+          path: '/todo',
+          queryParameters: {'q' : 'test'},
+      );
+      final restOperation = Amplify.API.get(restOptions: options);
+      final response = await restOperation.response;
+      print('GET call succeeded');
+      print(response.body);
+    } on ApiException catch (e) {
+      print('GET call failed: $e');
+    }
   }
-}
-```
+  ```
+
+</Block>
+
+<Block name="Developer Preview (Mobile, Web & Desktop)">
+
+  ```dart
+  Future<void> getTodo() async {
+    try {
+      final restOperation = Amplify.API.get(
+        'todo',
```

Comment:
What was the reason of droppong the / in the beginning at all places?

Reply:
good question: New version will work with/without "/" so wanted to make it more concise. Can change it back if folks find it confusing.

Reply:
I think if it works both ways, docs can stay as it is to enforce a c/leaner approach.